demos/handwritten_table_text_recognition_demo/python/README.md

```diff
@@ -0,0 +1,104 @@
+# Handwritten Chinese Table OCR Demo
+## Description
+This demo demonstrates using OpenCV and OpenVINO toolkit to recognize handwritten Chinese/Japanese tables and generate docx files with a GUI.
+
+
+## How It Works
+The demo uses the following model in the Intermediate Representation (IR) format:
+   * handwritten-japanese-recognition-0001
+   * handwritten-simplified-chinese-recognition-0001
+
+
+This demo workflow mainly has two parts:
+
+- The first part is the detection of the table. This part uses OpenCV to perform perspective transformation, and then detects the cells of the table.
+
+- The second part uses the cell images detected in the first part for handwritten text recognition with OpenVINO. It first reads a cell image and performs the preprocessing such as further binarization, resize and padding, and then the inference will start. After decoding the returned indexes into characters, the demo will display the predicted text. Optionally, you can generate the predicted text into a docx file.
+
+## Installation and dependencies
+### OS Requirement
```

Comment:
is there some platform specific behavior or you just test it using this OS?
I mean is it really should not work on Mac OS and Windows?

Reply:
It has **only** been tested in Ubuntu. It should work in Mac OS and windows.